app/javascript/packs/acuant_sdk_image_capture.js

```diff
@@ -0,0 +1,11 @@
+import { loadAndInitializeAcuantSdk } from '../app/acuant/sdk';
+
+document.addEventListener('DOMContentLoaded', () => {
+  window.ACUANT_SDK_INITIALIZATION_CREDS = document.querySelector(
```

Comment:
I would do this a little differently:

1. make these functions that on-demand load the values from the dom
2. dependency injection: update the `initializeAcuantSdk` to accept the creds as parameters like `initializeAcuantSdk(credentials, endpoint)` and then that will simplify testing and we won't need to worry about side effects

So like then it turns into something like:

```js
function loadAndInitializeAcuantSdk(credentials = null, endpoint = null) {
  credentials = (credentials || dom.loadCredentials());
  endpoint = (endpoint || dom.loadEndpoint());
}
```

Reply:
Ah interesting. Let me give that a shot.

Reply:
I just pushed a commit where I took a stab at what I think you're getting at here

Reply:
Yup! That's what I meant! But the tests could pass literal dummy values if they wanted to `loadAndInitializeAcuantSdk()` now which it looks like they're not. The benefit is that it would simplify the DOM setup in tests to just pass values directly

Reply:
like `loadAndInitializeAcuantSdk('aaa', 'bbb')`

Reply:
Oh just kidding I got it mixed up with `initializeAcuantSdk` -- we're good! 👍